City-of-Helsinki/helsinki-paatokset
# PP-292: Added load more functionality to policymaker page as well as listing below sectors

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-292

Added load more functionality to policymaker categories that have more than 50 rows.
Changed the way Viranhaltijat are presented
Added sectors for Lautakunnat ja johtokunnat

Also made a small fix for trustee page that the phone number is now visible

**How to test**

Checkout branch and run `drush cim -y; drush cr` and go to https://helsinki-paatokset.docker.so/fi/paattajat

- There should now be more accordions, Viranhaltijat have each their own accordion for each sector.
- If an accordion has more than 50 rows, there will be a load more -button, when all of the items are loaded, the button disappears (test also keyboard functionality)
- Lautakunnat ja johtokunnat should have the sector titles visible

Go to some trustee page and check that the phone number field works if it is filled.
